carma/launch/plugins.launch.py

```diff
@@ -79,9 +79,9 @@ def generate_launch_description():
 
     pure_pursuit_tuning_parameters = [vehicle_calibration_dir, "/pure_pursuit/calibration.yaml"]
 
-    carma_plugins_container = ComposableNodeContainer(
+    carma_lane_follow_plugins_container = ComposableNodeContainer(
         package='carma_ros2_utils',
-        name='carma_guidance_core_plugins_container',
+        name='carma_lane_follow_plugins_container',
         executable='carma_component_container_mt',
         namespace=GetCurrentNamespace(),
         composable_node_descriptions=[
```

Comment:
Any reason why we dont want to split inlanecruising and route following as well?